### DISCUSSION TITLE
Unable to change snapshot schedule

### DISCUSSION THREAD

**Discussion by jnizzle54** [wictorwilen/ring-timelapse#8] [Q&A]
Default of 15 mins is always used regardless if param is passed in to change it.  Also, changing value in Docker file then building and using new image with custom snapshot value does not appear to be working either.  The logs indicate snaphot attempts and specified interval but snapshot process fails.

**Comment by blueflame66** [comment]
I am having somewhat of the same issue.   For me the snapshot passes, but I want to change it from 15 min to 5 min if possible.   My environment variable is - CRON_SCHEDULE=*/5 * * * *     but the system ignores that and does it every 15 minutes.   Not sure if I have entered this correctly.

**Reply by elminster13** [reply]
Same here - I passed 10 mins tna
